src/webapp/components/sidebar/SideBar.tsx

```diff
@@ -37,7 +41,13 @@ export const SideBar: React.FC = () => {
     return (
         <CustomCard minheight="630px" padding="0 0 100px 0" data-test="test2">
             <HomeButtonWrapper>
-                <Button className="home-button" component={NavLink} to="/" exact={true} onClick={() => setModule("")}>
+                <Button
+                    className="home-button"
+                    component={NavLink}
+                    to="/"
+                    exact={true}
+                    onClick={() => resetCurrentModuleAccess()}
```

Comment:
`() => resetCurrentModuleAccess()  -> `resetCurrentModuleAccess`. Shorter, and keeps the prop immutable.

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/commit/c664c49dc8abcbeb0baeceac61d29b7d7275cb08